ui/src/pages/ApplicationDetailsPage/components/BranchesList/BranchCard.tsx

```diff
@@ -1,48 +1,57 @@
-import { Card, CardActionArea, CardContent, Skeleton, Stack, Typography } from '@mui/material'
+import { Card, CardActionArea, CardContent, Skeleton, Stack, Tooltip, Typography } from '@mui/material'
 import { ArtifactListItemReducedDTO } from '../../../../../autogenerated/client/backend'
 import { useNavigate } from 'react-router-dom'
 import moment from 'moment'
 import { AdjustOutlined, LayersOutlined, SvgIconComponent, SyncOutlined } from '@mui/icons-material'
 import GitBranch from '../../../../components/GitBranch/GitBranch'
-import { useState } from 'react'
+import { ReactElement, useState } from 'react'
 import HintIcon, { LoadingHintIcon } from './HintIcon'
 import InstallDialog from '../../../../components/Helm/InstallDialog'
 
-export default function BranchCard({ branch, version, artifact }: 
-{ branch: { name: string, pattern: string }, version: string, artifact: ArtifactListItemReducedDTO }) {
+export default function BranchCard({ branch, version, artifact, disabled = false }: 
+{ branch: { name: string, pattern: string }, version: string, artifact: ArtifactListItemReducedDTO, disabled?: boolean }) {
   const [ installDialogOpen, setInstallDialogOpen ] = useState<boolean>(false)
 
   const navigate = useNavigate()
 
   function onInstall() {
-    setInstallDialogOpen(false)
-    navigate('/workloads')
+    if (!disabled) {
+      setInstallDialogOpen(false)
+      navigate('/workloads')
+    }
   }
 
+  const AuthnWrapper = ({ children } : { children: ReactElement }) => disabled ? <Tooltip title='Log in to install this application'>{ children }</Tooltip> : <>{ children }</>
```

Comment:
Since you are not prompted directly to the login form, does it make sense to complete a little bit how to do so?

```suggestion
  const AuthnWrapper = ({ children } : { children: ReactElement }) => disabled ? <Tooltip title='Go to the extension settings and log in to install this application'>{ children }</Tooltip> : <>{ children }</>
```